ShopifySharp/Services/User/UserService.cs

```diff
@@ -24,15 +24,10 @@ public UserService(string myShopifyUrl, string shopAccessToken) : base(myShopify
         /// <summary>
         /// Gets all the users.
         /// </summary>
-        public virtual async Task<IEnumerable<User>> ListAsync(IUnpaginatedListFilter<User> filter = null)
+        public virtual async Task<IEnumerable<User>> ListAsync()
```

Comment:
As discussed, all ListAsync methods should allow passing in a filter so devs can use undocumented properties if they desire.